### DISCUSSION TITLE
vnote3.14  对于部分较长的md文档无法正常渲染

### DISCUSSION THREAD

**Discussion by 36huo** [tamlok/vnote#2208] [Q&A]
win10 + vnote3.14  对于部分较长（内部代码较多）的md文档会渲染失败，无法进入阅读模式。

使用了两台电脑（均为win10）不同的版本（3.13、3.14）均能够复现。 出问题的md文件长度也没有规律

出现问题的md 文件，请参考（如果删除最后一个代码块里的一些行，就会正常）

链接: https://pan.baidu.com/s/1g-CQLp9oydDHGizHZKA7dQ 提取码: 24qj 复制这段内容后打开百度网盘手机App，操作更方便哦

而且如果不是重新打开，那么渲染也一直正常（就是一直写或者粘贴， 切换编辑和阅读模式）。一旦关掉，再打开就不行了。 

不渲染时张这个样子
![](https://md.36huo.com/uploads/upload_786589e54c3d9ecb140d9fd3d10aa7a0.jpg)

![](https://md.36huo.com/uploads/upload_963b4233ab8cab2a34e7215757baf8e8.jpg)

同事的mac 版 3.13 能够正常渲染。

**Comment by 36huo** [comment]
点击调试器提示”DevTools was disconnected from the page“。  不知道哪里能看到渲染失败的日志么  

![](https://md.36huo.com/uploads/upload_fbe89ad1dfc9637f46946958b5c6a36f.jpg)

**Comment by tamlok** [comment]
If you find read mode not working on Windows, please download and try vnote-win-x64-qt5.15.2_v3.14.0.zip.

重新下载一个新版本的吧。
